test/validators/movie.test.js

```diff
@@ -10,15 +10,17 @@ describe('movie validator', () => {
 
     it('is required', () => {
       const payload = {};
-      const result = Joi.validate(payload, MovieValidator);
+      const result = Joi.validate(payload, MovieValidator.payload);
 
       expect(result.error.details[0].path).to.eql('title');
       expect(result.error.details[0].type).to.eql('any.required');
     });
 
     it('is less than 255 characters', () => {
-      const payload = { title: 'a'.repeat(260) };
-      const result = Joi.validate(payload, MovieValidator);
+      const payload = { 
+        title: 'a'.repeat(260) 
```

Comment:
nit: if we're testing that it's at most 255 characters we want to test with `256` to make sure it fails. Also I think a better test name might be `is at most 255 characters` or `is less than 256 characters` since `255` is a valid length.